### PR TITLE
use testcase Name as volume name  in e2e test cases

### PR DESCRIPTION
**Pull request by Madhu-1**
Signed-off-by: Madhu Rajanna <mrajanna@redhat.com>

### CODE REVIEW

**Comment by Madhu-1** [comment]
@prashanthpai @aravindavk @rishubhjain  please review

File: e2e/smartvol_ops_test.go [hunk 1]
```diff
@@ -55,7 +56,7 @@ func testSmartVolumeDistribute(t *testing.T) {
 
 func testSmartVolumeReplicate2(t *testing.T) {
 	r := require.New(t)
-	smartvolname := "sv2"
+	smartvolname := strings.Replace(t.Name(), "/", "-", 1)
```
Comment (prashanthpai) [inline e2e/smartvol_ops_test.go]: Why do you need the `strings.Replace` here ?

Comment (Madhu-1) [reply e2e/smartvol_ops_test.go]: Test name will become ```TestSmartVolume/Smartvol_Distributed_Volume```

we cannot use / in volume names. need to replace / with -

Comment (phlogistonjohn) [reply e2e/smartvol_ops_test.go]: What might be nice is a short helper function, something like testVolName(t) to make it easier for future test writers to do the right thing.

Comment (Madhu-1) [reply e2e/smartvol_ops_test.go]: done

**Comment by Madhu-1** [comment]
resolved merged conflict. @aravindavk can you please review this.

**Pull request merged**